.travis.yml

```diff
@@ -48,4 +48,10 @@ jobs:
       go: 1.8.x
 
 notifications:
-  irc: "chat.freenode.net#cri-o"
+    irc:
+        channels:
+            -  "chat.freenode.net#cri-o"
+        template:
+            - "%{author}, %{repository}#%{build_number} (%{branch} - %{commit} : %{author}): %{message}"
+            - "%{author}, ^^^^^ changes: %{compare_url}"
+            - "%{author}, ^^^^^ details : %{build_url}"
```

Comment:
what's new in this that we don't already have:
```
11:04                   travis-ci : kubernetes-incubator/cri-o#3371 (master - a3a0bb5 : Daniel J Walsh): The build passed.
11:04                   travis-ci : Change view :
                                    https://github.com/kubernetes-incubator/cri-o/compare/030b462e3586...a3a0bb5e5538
11:04                   travis-ci : Build details : https://travis-ci.org/kubernetes-incubator/cri-o/builds/274193731
```

Reply:
It "mentions" the author name in each message.  @rhatdan requested this, I blame him 😄

Reply:
is the "author" the github handle? cause we do already get the author here 
```
11:04                   travis-ci : kubernetes-incubator/cri-o#3371 (master - a3a0bb5 : Daniel J Walsh): The build passed.
```

Reply:
I was hoping to get IRC name somehow in there.

So if this said Daniel J Walsh (dwalsh)
Antonio Murdaca (runcom)
Chris Evich (cevich)
...


Then when it shows up in IRC, It will highlight that someone/thing mentioned my name  and I will look for it.

Reply:
Alright I doubt the current patch could get IRC nick, it can just grab nick from the Github handle...

Reply:
Aye, unf. there's no way to do that using the built-in irc notifier because it doesn't accept any non-canned substitution tokens 😦 (so no lookup tables of author vs IRC nick).  You'd have to add your own highlighting in your IRC client, for your github username.

If the author-to-nick mapping piece is important, we'll need to find and haul-in some IRC notification script/bot and run it at the end of testing.  It's doable, and portable over to papr (if/when we get there), but it's not as simple as using the built-in thing.  Is this the direction y'all'd prefer?

Reply:
Well that is my goal so that when something I launch gives me a notification in IRC that it is finished.

Reply:
Okay, I've opened a card for this:  https://trello.com/c/3zei9T1A